src/services/marking-service.ts

```diff
@@ -1,8 +1,40 @@
 import { MarkingVerificationType } from '../api/marking/marking.utils';
 import { MarkingRepository } from '../repositories/marking-repository';
-import { InternalService } from './base-service';
+import { Service } from './base-service';
+import { ItisService } from './itis-service';
+
+/**
+ * Marking Service
+ *
+ * @export
+ * @class MarkingService
+ * @implements Service
+ */
+export class MarkingService implements Service {
+  repository: MarkingRepository;
+  itisService: ItisService;
+
+  /**
+   * Construct MarkingService class.
+   *
+   * @param {MarkingRepository} repository - Repository dependency.
+   * @param {ItisService} itisService - Itis service dependency.
+   */
+  constructor(repository: MarkingRepository, itisService: ItisService) {
+    this.repository = repository;
+    this.itisService = itisService;
+  }
+
+  /**
+   * Instantiate MarkingService and inject dependencies.
+   *
+   * @static
+   * @returns {MarkingService}
+   */
+  static init(): MarkingService {
+    return new MarkingService(new MarkingRepository(), new ItisService());
+  }
```

Comment:
question: Is this just for convenience? So if you don't want to overwrite anything by calling the constructor, you can just call this?

Reply:
I wonder what would happen if you called:

`new MarkingService(a, b).init()`?

Reply:
Ah its static, nvm!

Reply:
Yea just for convenience, anticipating some of the services (critter / bulk) to have quite a few dependencies. I kinda like it for the benefits of DI in regards to testing and the simplicity of instantiating (like if you just directly imported the dependencies)